ansible_rulebook/job_template_runner.py

```diff
@@ -106,7 +106,7 @@ async def _get_job_template_id(self, name: str, organization: str) -> int:
                     break
 
         raise JobTemplateNotFoundException(
-            f"{name} in organization {organization}"
+            f"Job template {name} in organization {organization} does not exist"  # noqa: E501
```

Comment:
@bzwei This can be changed to

```
raise JobTemplateNotFoundException(
            (   
                f"Job template {name} in organization "
                f"{organization} does not exist"
            )   
        ) 
```

Reply:
@mkanoor You can make a code suggestion, so the PR owner can accept it directly.

Reply:
```suggestion
            ( 
                 f"Job template {name} in organization " 
                 f"{organization} does not exist" 
            )
```